ADALiOS/ADALiOS/ADAuthenticationContext.h

```diff
@@ -273,6 +274,25 @@ typedef void(^ADAuthenticationCallback)(ADAuthenticationResult* result);
              extraQueryParameters: (NSString*) queryParams
                   completionBlock: (ADAuthenticationCallback) completionBlock;
 
+/*! Follows the OAuth2 protocol (RFC 6749). The behavior is controlled by the promptBehavior parameter on whether to re-authorize the
+ resource usage (through webview credentials UI) or attempt to use the cached tokens first.
+ @param resource: the resource for whom token is needed.
+ @param clientId: the client identifier
+ @param redirectUri: The redirect URI according to OAuth2 protocol
```

Comment:
:+1: A number of the other document blocks have this problem, and also reference now-gone parameters.